js/index.js

```diff
@@ -1,6 +1,49 @@
 // Iteration 1: Names and Input
 //
+let hacker1 = 'Jose';
+let hacker2 = 'Farzin';
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    'The driver has the longest name, it has ' + hacker1.length + ' characters.'
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    'It seems that the navigator has the longest name, it has ' +
+      hacker2.length +
+      ' characters.'
+  );
+} else {
+  console.log(
+    'Wow, you both have equally long names, ' + hacker1.length + ' characters!'
+  );
+}
 
 // Iteration 3: Loops
+
+console.log(hacker1.toUpperCase().split(''));
+
+console.log(hacker2.split('').reverse().join(''));
+
+/* if ('hacker2'.localeCompare('hacker1') === 1) {
+    console.log("The driver's name goes first.")
+  }else if ('hacker1'.localeCompare('hacker2') === -1){
+    console.log("Yo, the navigator goes first definitely.")
+      }else {
+        console.log("What?! You both have the same name?")
+                 }*/
+
+let arr = [hacker1, hacker2];
+arr.sort();
```

Comment:
Here is the same thing as above, pay attention that not always `.sort()` behaviors as you think it would, see the examples => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort